src/sentry/static/sentry/app/views/settings/account/accountDetails.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {addErrorMessage, addSuccessMessage} from 'app/actionCreators/indicator';
+import AsyncView from 'app/views/asyncView';
+import AvatarChooser from 'app/components/avatarChooser';
+import Form from 'app/views/settings/components/forms/form';
+import JsonForm from 'app/views/settings/components/forms/jsonForm';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import accountDetailsFields from 'app/data/forms/accountDetails';
+import accountPreferencesFields from 'app/data/forms/accountPreferences';
+import {APIRequestMethod} from 'app/api';
+import {t} from 'app/locale';
+
+const ENDPOINT = '/users/me/';
+
+class AccountDetails extends AsyncView {
+  getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
+    return [['user', ENDPOINT]];
+  }
+
+  renderBody() {
+    const {user} = this.state;
+    const {location} = this.props;
+
+    const formCommonProps: Partial<Form['props']> = {
```

Comment:
@priscilawebdev From what I can tell, almost all props on `Form` are optional anyways. Well, except for the callbacks. I am wondering why `onFieldChange` is required. Based off the usage, I think all the callbacks should be optional. That way you wouldn't need to use `Partial` here. Not big deal though :)

Reply:
Hi @scefali! yes I agree with you...I will check and maybe change it to be optional...